src/onegov/org/models/organisation.py

```diff
@@ -237,3 +242,13 @@ def title_lines(self):
 
     def excluded_person_fields(self, request):
         return [] if request.is_logged_in else self.hidden_people_fields
+
+    @property
+    def event_filter_fields(self):
+        return flatten_event_filter_fields_from_definition(
+            self.event_filter_definition)
+
+
+@lru_cache(maxsize=1)
```

Comment:
I would not set a maxsize this small, since this cache is shared between all tenants, you can do it if you move this cache to a method on the app, because then the maxsize means you can only have one of these per app and the app sticks around, otherwise you will need to go for something larger, like maybe 64/128?

Reply:
Makes sense